keras_cv/models/stable_diffusion/image_encoder.py

```diff
@@ -35,13 +35,13 @@ def __init__(self, img_height=512, img_width=512, download_weights=True):
                 PaddedConv2D(128, 3, padding=1),
                 ResnetBlock(128),
                 ResnetBlock(128),
-                PaddedConv2D(128, 3, padding=1, strides=2),
+                PaddedConv2D(128, 3, padding=((0, 1), (0, 1)), strides=2),
```

Comment:
Now that we think we have the forward pass correct, let's add a forward pass numerical test. (one of the disabled-by-default tests under stable_diffusion_tests is fine with me)